lib/dynflow/persistence_adapters/sequel_migrations/006_fix_data_length.rb

```diff
@@ -0,0 +1,15 @@
+Sequel.migration do
+  up do
+    alter_table(:dynflow_steps) do
+      if Sequel::Model.db.database_type == :mysql
```

Comment:
@db should be accessible and already set to the db being migrated inside the alter_table blocks